python/pyspark/ml/classification.py

```diff
@@ -17,6 +17,8 @@
 
 import operator
 import sys
+import uuid
```

Comment:
Ohhhh sorry it was my mistake. It doesn't crash because `uuid ` is imported via `from pyspark.ml.util import *`.

Reply:
Okay, but let's just backport since it's open.